setup.py

```diff
@@ -37,7 +39,10 @@ def load_requirements(*requirements_paths):
 
 README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
 REQUIREMENTS = load_requirements('requirements.txt')
-REQUIREMENTS.append('Django>=1.8,<2')
+if sys.version_info.major < 3:
+    REQUIREMENTS.append('Django>=1.8,<2')
+else:
+    REQUIREMENTS.append('Django>=1.8,<3')
```

Comment:
Hmm.  How about instead putting two entries for Django in `requrements.txt` with environment markers which make them only apply under different Python versions?  Like `Django>=1.11,<2 ; python_version == "2.7"`.

Reply:
implemented in the **avoid duplication of Django version in setup.py & tox.ini** commit